cli/cmd/deploy/deploy.go

```diff
@@ -118,13 +128,9 @@ func DeployCmd(cfg *config.Config) *cobra.Command {
 				return nil
 			}
 
-			userLoginSuccess := false
 			silentGitFlow := false
 			// If user is not authenticated, run login flow
 			if !cfg.IsAuthenticated() {
-				info.Println("Please log in or sign up for Rill. Opening browser...")
-				time.Sleep(2 * time.Second)
-				silentGitFlow = true
```

Comment:
It looks like `silentGitFlow` does not get set to `true` here after the refactor. Is that intended?

Reply:
Thanks. Its a mistake it needs to be set to `true`.